src/Tribe/Assets.php

```diff
@@ -393,6 +393,20 @@ public function register() {
 				],
 			]
 		);
+
```

Comment:
Should we load the asset like we do for the upgrade tab only if the upsell is active?

https://github.com/the-events-calendar/the-events-calendar/blob/b889b8762d5fd2f38885326653ceebf0c897a5eb/src/Tribe/Main.php#L1311

Reply:
Ooh yeah, it's better to load them only when the upsell is active. I've made the changes.